pkgs/development/python-modules/drawille/default.nix

```
@@ -0,0 +1,19 @@
+{lib, buildPythonPackage, fetchPypi}:
+
+buildPythonPackage rec {
+  pname = "drawille";
+  version = "0.1.0";
```

```suggestion
  version = "0.1.0";
  format = "setuptools";
```

Set an explicit format, `setuptools` for `setup.py` based builds. Same for the other packages.